about/team/index.md

```diff
@@ -34,3 +34,15 @@ ADS research associates are:
 The late [Stephen S. Murray](https://www.cfa.harvard.edu/news/stephen-s-murray-high-energy-astrophysicist-dies-age-70) served as the Principal Investigator of the project from its inception in 1993 until his passing in August 2015.
 
 If you have comments or questions about the ADS, you are welcome to contact any of us directly, although the preferred way to get in touch with ADS staff is through our support email ([adshelp [at] cfa.harvard.edu](mailto:adshelp@cfa.harvard.edu)), which guarantees a timely reply to your inquiry.
+
+<div>
+
+<div style="float:left;">
+  <div class="floated_img">
+   <img src="{{ site.baseurl }}/about/team/img/Team_photo_Mar2024.jpeg" alt="Team photo from March 2024 Retreat" height="325" width="1000" alt="Image" style="float: left; margin: 4px 10px 0px 0px; border: 1px solid #000000;">
```

Comment:
You've got two `alt` tags here (also for the `scixabout` version). I'd remove the generic one - I'm not sure if screen readers can handle two of these for a single photo?

Reply:
Also, I'd resize the photo - if you try viewing it on a cell phone-sized screen, things get wonky. I think the smallest viewport is this one: https://github.com/adsabs/adsabs.github.io/blob/master/_sass/bootstrap/_variables.scss#L347 (it's 720px), so maybe set the width of the photo to 700 or 710?

Reply:
Changing the width to 710px makes the website version look nice since it the image is not wider than the text, but the image still overflows the text in the mobile version

![image](https://github.com/adsabs/adsabs.github.io/assets/35502000/937acd39-a0fd-4257-9cd5-34be42ead2b1)
![image](https://github.com/adsabs/adsabs.github.io/assets/35502000/f6813939-e711-4d05-b96c-0607b706b89e)